experimental/traffic-portal/src/app/core/invalidation-jobs/invalidation-jobs.component.html

```diff
@@ -11,32 +11,39 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
-<tp-header title="{{deliveryservice ? deliveryservice.displayName : 'Loading'}} - Content Invalidation Jobs"></tp-header>
-<ul>
-	<li class="invalidation-job" *ngFor="let j of jobs">
-		<code>{{j.assetUrl}}</code> (active from <time [dateTime]="j.startTime">{{j.startTime | date:'medium'}}</time> to <time [dateTime]="endDate(j)">{{endDate(j) | date:'medium'}})</time>
-		<button
-			mat-icon-button
-			type="button"
-			color="accent"
-			title="Edit this content invalidation job"
-			aria-label="Edit this content invalidation job"
-			[disabled]="j.startTime <= now"
-			(click)="editJob(j)"
-		>
-			<fa-icon [icon]="editIcon"></fa-icon>
-		</button>
-		<button
-			mat-icon-button
-			type="button"
-			color="warn"
-			title="Delete this content invalidation job"
-			aria-label="Delete this content invalidation job"
-			[disabled]="isInProgress(j)"
-			(click)="deleteJob(j.id)"
-		>
-			<fa-icon [icon]="deleteIcon"></fa-icon>
-		</button>
-	</li>
-</ul>
-<button mat-fab type="button" id="new" title="Create a new content invalidation job for this Delivery Service" (click)="newJob()"><fa-icon [icon]="addIcon"></fa-icon></button>
+<mat-card>
+	<ul>
+		<li class="invalidation-job" *ngFor="let j of jobs">
+			<div>
+				<div class="invalidation-info">
+					<span>{{j.assetUrl}} (active from <time [dateTime]="j.startTime">{{j.startTime | date:'medium'}}</time> to <time [dateTime]="endDate(j)">{{endDate(j) | date:'medium'}})</time></span>
+				</div>
+				<div class="spacer"></div>
```

Comment:
This div is unnecessary; the class applies no styling and it appears to exist simply to take up space. Just set the parent element's `justify-content` to `space-between` to achieve the same look without needing useless elements.